### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Hixie**
(If this doesn't land before we move the chat around, i'll update that part)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,6 +23,23 @@ and submitting pull requests for the Flutter framework. For
...
+1. Join the Gitter channel.
+
+2. Help out as described in our wiki: https://github.com/flutter/flutter/wiki/Triage
+   You won't be able to add labels at first, so instead start by trying to
+   do the other steps, e.g. trying to reproduce the problem and asking for people to
+   provide enough details that you can reproduce the problem, pointing out duplicates,
+   and so on. Chat on the gitter channel to let us know what you're up to!
+ 
+3. Once you've been doing this for a while, someone will invite you to the flutter-hackers
+   team on GitHub and you'll be able to add labels too.
```
Comment (Piinks) [inline CONTRIBUTING.md]: Does being a part of flutter-hackers also provide the ability to approve and merge?

Comment (Hixie) [reply CONTRIBUTING.md]: i'll address this in my update today

Comment (Hixie) [reply CONTRIBUTING.md]: documented in https://github.com/flutter/flutter/wiki/Contributor-access

**Comment by xster** [comment]
LG

**Review by Piinks — APPROVED** [review]
LGTM!

**Pull request merged**